src/reviewerQueue.js

```diff
@@ -100,8 +100,11 @@ HUBOT_GITHUB_REVIEWER_MAIL_MAP: ${ghReviewerEmailMap}\
 
     const stats = robot.brain.get(STATS_KEY) || {};
 
-    // (re)initialize stats if necessary
-    if (!stats['reviewers'] || !_.isEqual(stats['reviewers'], reviewers)) {
+    // (re)initialize stats if reviewers is empty or the members have changed.
+    if (
+      !stats['reviewers'] ||
+      !_.isEqual(_.sortBy(stats['reviewers'], 'login'), _.sortBy(reviewers, 'login'))
```

Comment:
This isn't quite right either - what if other fields besides login (that we don't care about) have changed?

I think you need to compare the sorted list of (reviewer)=>reviewer.login.

(Btw, I'm sure it was an artifact of the coffeescript conversion, but stats['reviewers'] could be stats.reviewers.)